### PR TITLE
Update reduced-form-specification.md

### PR DESCRIPTION
**Pull request by azharhsain**
Addressed all the reviewer comments for this file.

### CODE REVIEW

File: tutorial-content/content/reduced-form-specification.md [hunk 1]
```diff
@@ -61,65 +61,16 @@ Temperature relationships are often preferred in climate impacts research, becau
 
 ### *Humidity*
 
-There are mainly three metrics for humidity measurements: absolute, relative (often "RH"), and specific. Absolute humidity describes the water content of air,
+There are mainly three metrics for humidity measurements: absolute, relative (often "RH"), and specific (very similar to absolute). Absolute humidity describes the water content of air,
```
Comment (jrising) [inline tutorial-content/content/reduced-form-specification.md:64]: I would put this in the sentence explaining specific humidity.

**Review by jrising — CHANGES_REQUESTED** [review]
A couple comments on this. Thanks for the edits so far!

File: tutorial-content/content/reduced-form-specification.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Temperature relationships are often preferred in climate impacts research, becau
 1. **$T_{min}$, $T_{max}$:**  Many socioeconomic processes are more
     sensitive to extreme temperatures than to variation in the
     average. This is also useful when temperature variation is
-    large, leading to significant differences in the cold end and hot end responses. These are important metrics when heterogeneity within time units matters, and may better capture heat waves and cold spells. Also, note that $T_{min}$ better reflects nighttime temperatures while $T_{max}$ better reflects daytime temperatures. Not all datasets include $T_{min}$ or $T_{max}$. 
+    large, leading to significant differences in the cold end and hot end responses. These are important metrics when heterogeneity within time units matters, and may better capture heat waves and cold spells. Also, note that $T_{min}$ better reflects nighttime temperatures while $T_{max}$ better reflects daytime temperatures. Not all datasets include $T_{min}$ or $T_{max}$, which are measured at a daily frequency.
```
[the remark this answers is not in the document]
Comment (azharhsain) [reply tutorial-content/content/reduced-form-specification.md:47]: Done!

File: tutorial-content/content/reduced-form-specification.md (hunk 1 above)
Comment (azharhsain) [reply tutorial-content/content/reduced-form-specification.md:64]: Done!

File: tutorial-content/content/reduced-form-specification.md [hunk 3]
```diff
@@ -223,20 +168,70 @@ happens at the office level only.  Estimating grid-level effects will
...
...is often used as a control since it is correlated with temperature. However, the strength and direction of this correlation can vary significantly by region and time of year (see e.g. [Trenberth et al. 2005](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2005GL022760), with implications for collinearity). Furthermore, the same care should be taken when inserting precipitation into a model as any other weather or social variable - what is its expected role? In what form should the data be? etc. Precipitation affects society differently at different spatiotemporal scales - annual precipitation may be useful for studying snowpack trends, drinking water supply, or the effect of droughts on agriculture; maximum precipitation rates may be the relevant metric for flood damages or crop failures. Moreover, precipitation can be much more approximated by a Normal disribution when aggregated in time, e.g. monthly precip. Remember that though means and extremes may be correlated, it's still possible to have a record storm in an unnaturally dry year, or an unnaturally wet year without heavy precipitation. As a result, different metrics of precipitation are often used (incomplete list):
```
[the remark this answers is not in the document]
Comment (azharhsain) [reply tutorial-content/content/reduced-form-specification.md:187]: Skipped adding the precipitation graph as it seems unnecessary.  Let me know

[the remark this answers is not in the document]
Comment (jrising) [reply tutorial-content/content/reduced-form-specification.md:187]: I don't think we can skip things. It's a review, and reviewers get upset when you ignore their comments. We could write up a clear justification for why we didn't do it, but it's easier in this case to just do it.

I was collecting some precip. data from ERA5, so I decided to do this for you. I'll email you the result, and you can write it up.

**Pull request merged**